src/test/java/uk/gov/hmcts/cmc/claimstore/utils/CaseDetailsConverterTest.java

```diff
@@ -37,34 +40,66 @@
     @Mock
     private NonWorkingDaysCollection nonWorkingDaysCollection;
 
-    @Before
-    public void setup() {
-        caseDetailsConverter = new CaseDetailsConverter(caseMapper,
-            JsonMapperFactory.create(),
-            new WorkingDayIndicator(publicHolidaysCollection, nonWorkingDaysCollection),
-            33);
-    }
+    @Nested
+    class CtscSDisabledTests {
 
-    @Test
-    public void convertsCaseDetailsToCCDCase() {
-        CaseDetails caseDetails = successfulCoreCaseDataStoreSubmitResponse();
-        CCDCase ccdCase = caseDetailsConverter.extractCCDCase(caseDetails);
-        assertThat(ccdCase.getId()).isEqualTo(caseDetails.getId());
-        assertThat(ccdCase.getState()).isEqualTo(caseDetails.getState());
-    }
+        @BeforeEach
+        void setup() {
+            caseDetailsConverter = new CaseDetailsConverter(caseMapper,
+                JsonMapperFactory.create(),
+                new WorkingDayIndicator(publicHolidaysCollection, nonWorkingDaysCollection),
+                33,
+                false);
+        }
+
+        @Test
+        void convertsCaseDetailsToCCDCase() {
+            CaseDetails caseDetails = successfulCoreCaseDataStoreSubmitResponse();
+            CCDCase ccdCase = caseDetailsConverter.extractCCDCase(caseDetails);
+            assertThat(ccdCase.getId()).isEqualTo(caseDetails.getId());
+            assertThat(ccdCase.getState()).isEqualTo(caseDetails.getState());
+        }
+
+        @Test
+        void convertsCaseDetailsToClaim() {
+            CaseDetails caseDetails = successfulCoreCaseDataStoreSubmitResponse();
+            Claim claim = caseDetailsConverter.extractClaim(caseDetails);
+            assertThat(claim.getId()).isEqualTo(caseDetails.getId());
+            assertThat(claim.getState().getValue()).isEqualTo(caseDetails.getState());
+        }
 
-    @Test
-    public void convertsCaseDetailsToClaim() {
-        CaseDetails caseDetails = successfulCoreCaseDataStoreSubmitResponse();
-        Claim claim = caseDetailsConverter.extractClaim(caseDetails);
-        assertThat(claim.getId()).isEqualTo(caseDetails.getId());
-        assertThat(claim.getState().getValue()).isEqualTo(caseDetails.getState());
+        @Test
+        void convertsCaseDetailsToClaimWithIntentionToProceedDeadline() {
+            CaseDetails caseDetails = successfulCoreCaseDataStoreSubmitResponse();
+            Claim claim = caseDetailsConverter.extractClaim(caseDetails);
+            assertThat(claim.getIntentionToProceedDeadline()).isNotNull();
+        }
+
+        @Test
+        void convertsCaseDetailsToClaimWithResponseMethod() {
```

Comment:
Please change name to include "without" instead or "with

Reply:
Done